helm/prometheus-operator-app/Chart.yaml

```diff
@@ -34,4 +33,17 @@ annotations:
       url: https://github.com/prometheus-community/helm-charts
     - name: Upstream Project
       url: https://github.com/prometheus-operator/kube-prometheus
-  application.giantswarm.io/values-schema: https://raw.githubusercontent.com/giantswarm/prometheus-operator-app/v[[ .Version ]]/helm/prometheus-operator-app/values.schema.json
```

Comment:
I think you should add this back

Reply:
Check https://github.com/giantswarm/giantswarm-test-catalog/commit/c4778c8a00afeb4b41462de04f57ac72a048dbfe#diff-08c65c1b5ef8384ef0200558777c52f4110afcbbacc880ac5375d555ca97ee3eR27005

app-build-suite adds some of these annotations automatically